_timew

```diff
@@ -87,7 +87,7 @@ __timew_tags() {
 
     vals=$(sed '1,1d' ~/.timewarrior/data/tags.data | \
         sed '$d' | \
-        sed -E "s;^[[:space:]]+\"([[:space:][:alnum:]\.-]+)\":.*$;\1;g")
+        sed -E "s;^[[:space:]]+\"([[:space:][:alnum:]\.-_]+)\":.*$;\1;g")
```

Comment:
I think the regex is understanding this hyphen placement as a range indicator. I think swapping the positions does what you want.

```suggestion
        sed -E "s;^[[:space:]]+\"([[:space:][:alnum:]\._-]+)\":.*$;\1;g")
```

Reply:
Yes - good spot! I can confirm that this works for me with tags that contain both underscores and hyphens.